src/components/VTable/VTable.vue

```diff
@@ -75,7 +75,7 @@
         </thead>
         <tbody>
           <slot v-bind="{ items: cItems, ...$data, perPage }">
-            <tr v-for="(item, index) in cItems" :key="item.id">
+            <tr v-for="(item, index) in cItems" :key="item.id" @click="$emit('rowClick', item)">
```

Comment:
Think the event name should be "click:row"?

Reply:
I'm not even sure you can name it that. But thinking it might follow Vue standards more closely.

Reply:
I like that more. I don't see why it wouldn't work in Vue. I'll do a test and resubmit my request if it works.